infra/bitcoin/run.sh

```diff
@@ -28,6 +29,6 @@ do
     /app/bin/bitcoin-cli -regtest generatetoaddress 1 $ADDRESS
     sleep 5
     # send tx to own address while paying fee to the miner
-    /app/bin/bitcoin-cli -regtest sendtoaddress $ADDRESS 1 "" "" true
+    /app/bin/bitcoin-cli -regtest -named sendtoaddress address=$ADDRESS amount=0.1 subtractfeefromamount=false fee_rate=1
```

Comment:
Did the amount we're sending ourselves get reduced from 1 to 0.1?

Reply:
I was just checking different amounts. It doesn't matter though, this transaction is there to only simulate mining, so the amount is irrelevant as long as it covers fees

Reply:
Doesn't that get sent to the primary account we use for testing?

Reply:
I don't follow, `$ADDRESS` is the main account that we are using. Then we simulate mining wherein we generate blocks to `$ADDRESS` and send some amount `x` from `$ADDRESS` to itself.

Reply:
Ah ok, got it.